### DISCUSSION TITLE
How to change collection of the volumes?

### DISCUSSION THREAD

**Discussion by infinitisno** [seaweedfs/seaweedfs#4783] [Q&A] [answered]
Hello.

Initially launched master with option `-defaultReplication 001`,  and launched two volume servers (hdd and nvme) without `-disk` option. 

Then added another `nvme`, but now launched 2 `nvme` volumes with `-disk=nvme` option,
and one `hdd` volume with `-disk=hdd` option.

Got error - seaweedfs could not create new volumes for replication `001`....

Tried various things, and finally changing default collection for the filer did trick, but now I have bunch of volumes in default collection (colection name = ""), and new volumes in the new collection.

Is it possible to change old volumes from default collection to the new collection?

**Comment by duckman** [comment]
I've ran into this as well. I got lucky and spotted it before moving a bunch of data. Is there a reason there isn't a way to set the collection to a new collection when doing a collection.delete?

**Comment by chrislusf** [comment, marked as answer]
To change a volume's collection:
1. Stop the volume server
2. Rename the `xxx_[volume_id].dat` file to `yyy_[volume_id].dat`, and similarly to the `.idx` file.
3. Restart the volume server.
